### ISSUE TITLE
Image buttons should have complete alt text

### ISSUE THREAD

**Issue by sergiobaptista** [TBD54566975/ssi-service#43]
https://github.com/TBD54566975/ssi-service/blob/main/README.md

In all cases, the alt includes only the text in the gray sections. Make these

godoc ssi-service

go version 1.17.6

license Apache 2

issue 13 open 

ssi-service-ci passing

**Comment by sergiobaptista** [comment]
@angiejones 
the "ssi-service-ci" has an alt text saying only "Push". I think we should follow the strategy from the SSI-SDK page and change it to "ssi-service-ci status"
